src/segment-loader.js

```diff
@@ -1325,6 +1326,11 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
 
     this.logger_(segmentInfoString(segmentInfo));
+    segmentInfo.bytes.transmux = segment.uri === 'media_00001.ts';
```

Comment:
This would just use isFmp4 detection from the tba branch, but hard coding is easier to show an example.

Reply:
This is basically the hack here, as adding ` transmux` property to the bytes passed to appendBuffer is really strange.